CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: Allow pickup in Ghavoran Elun Transport Access by charging speed via navigation room
 - Changed: Help solver by adding Morph Ball requirment on connections to event to flip the spinner in Ghavoran Flipper Room
 - Changed: Shooting occluded objects requires at least Intermediate Knowledge
+- Changed: Fixed the node connection from Cataris / Door to Dairon Transport Access - Past Magnet Floor to Cataris / Door to Dairon Transport Access - Pickup (Energy Tank)
```

Comment:
I'm not sure how to word it, but for one should probably remove the fixed part. This isn't really "fixing" anything, just changing a connection. Can probably word it something like:

"Changed: Door to Total Recharge Station in Cataris - Thermal Device Room North connects directly to the pickup."

After all, the connection you changed is from Total Recharge, not Dairon Transport.